easybuild/easyconfigs/h/HDF/HDF-4.2.13-intel-2017a-no-netcdf.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'HDF'
+version = '4.2.13'
+versionsuffix = '-no-netcdf'
+
+homepage = 'http://www.hdfgroup.org/products/hdf4/'
+description = """HDF (also known as HDF4) is a library and multi-object file format for storing and managing data
+ between machines.  This version suppresses the netcdf api, that gives issues with some applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.hdfgroup.org/ftp/HDF/releases/HDF%(version)s/src/']
+checksums = ['be9813c1dc3712c2df977d4960e1f13f20f447dfa8c3ce53331d610c1f470483']
+
+builddependencies = [
+    ('flex', '2.6.3'),
+    ('Bison', '3.0.4'),
+]
+dependencies = [
+    ('Szip', '2.1'),
+    ('libjpeg-turbo', '1.5.1'),
```

Comment:
Bump to libjpeg-turbo 1.5.2? required easyconfig is already there...

Reply:
Done